### ISSUE TITLE
Issue importing pymanopt due to numpy config

### ISSUE THREAD

**Issue by YanniPapandreou** [pymanopt/pymanopt#167]
Hi,

Thanks a lot for the great work that you're doing with Pymanopt! 

I am running into an error importing Pymanopt, though it was working a few weeks ago.

I run the following line:

```python
import pymanopt
```

and I receive the following error:

```python
AttributeError: module 'numpy.distutils.__config__' has no attribute 'blas_opt_info'
```

The environment/version info I am using are:

- system: ubuntu 20.04
- Python 3.8.12
- Pymanopt: 0.2.5
- Numpy: 1.22.3

I have tried a suggestion I found online to manually set the option via:

```python
import numpy
numpy.distutils.__config__.blas_opt_info = numpy.distutils.__config__.blas_ilp64_opt_info

import pymanopt
```

I still get an error but the error is now:

```python
AttributeError: partially initialized module 'theano' has no attribute 'graph' (most likely due to a circular import)
```

Any help would be much appreciated :)

**Comment by semlen** [comment]
Hi,

I had the same error when I tried calling pymanopt in R via the reticulate package. For me, it helped forcing the Python version to 3.8.10. I'm using the following configurations and it works now:
-    system: ubuntu 20.04
-    Python 3.8.10
-    Pymanopt: 0.2.5
-    Numpy: 1.22.3
Hope this helps you.

**Comment by nkoep** [comment]
Hey @YanniPapandreou :wave: Could you maybe try again with the most recent version of pymanopt on pypi to confirm that this issue remains?

**Comment by YanniPapandreou** [comment]
Hi,

@semlen thanks for this, this helped back then - sorry for the delay in replying.

@nkoep I have tried again with the version from pypi and it seems to no longer be an issue. Just to clarify, I switched OS since March so my versions and system are all different. I created a virtual environment and installed pymanopt from pypi. Here is my current system info:

- system: Manjaro Linux x86_64
- Python: 3.10.4
- Pymanopt: 1.0.0
- Numpy: 1.22.4

Thanks.
Yanni
